main.css

```diff
@@ -114,3 +115,328 @@ ul.social {
 li.social-icon {
   display: flex;
 }
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  position: static;
+  width: 390px;
+  height: 600px;
+  left: calc(50% - 327px / 2);
+  top: 114px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 88px 20px;
```

Comment:
- [ Optional] I think you should set the `margin: 88px auto;`, this will make your cards be adjusted equally on the vertical axis.

Reply:
Thanks I will do that in my next commit